plugins/modules/inventory_intent.py

```diff
@@ -1127,14 +1163,21 @@ def resync_devices(self):
         device_ips = self.config[0].get("ip_address", [])
 
         if not device_ips:
-            msg = "Cannot perform the Resync operation as device's are not present in Cisco DNA Center"
-            self.status = "failed"
-            self.msg = msg
-            self.log(msg)
+            self.msg = "Cannot perform the Resync operation as device's {0} are not present in Cisco DNA Center".format(str(device_ips))
+            self.status = "success"
+            self.result['changed'] = False
+            self.log(self.msg)
             return self
 
-        device_ids = self.get_device_ids(device_ips)
+        ap_devices = self.check_ap_devices(device_ips)
+        self.log("AP Devices from the playbook input are : {0}".format(str(ap_devices)))
 
+        if ap_devices:
+            for ap_ip in ap_devices:
+                device_ips.remove(ap_ip)
+            self.log("Following devices {0} are AP so can't perform resync operation.".format(str(ap_devices)))
```

Comment:
if 


if  f string is supported then: (Also add a comma between AP and so
`self.log(f"Following devices {ap_devices} are AP, so can't perform the resync operation.")
`Otherwise:
`self.log("Following devices {0} are AP so can't perform resync operation.".format(str(ap_devices)))`

Reply:
f string is not supported, added comma after AP.